### ISSUE TITLE
Haniffa - Covid19PBMC E-MTAB-10026

### ISSUE THREAD

**Issue by Wkt8** [ebi-ait/hca-ebi-wrangler-central#261]
**Primary Wrangler:**
Wei Kheng Teh

**Secondary Wrangler:**
Marion

**Associated files:**
Pre-print: https://www.medrxiv.org/content/10.1101/2021.01.13.21249725v1.full
Google Drive: https://drive.google.com/drive/folders/1uoTWDLj57LTM9ZT2ceUQU8kF5QeWJmTM
Data available in the Covid19 Portal: https://www.covid19cellatlas.org/index.patient.html
Processed data available at E-MTAB-10026: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-10026/

**Key Events**

- [x] Send initial instructions about contributing, including questionnaire and T&Cs
- [x] Receive project questionnaire
- [x] Assign the Primary Wrangler
- [ ] Receive T&C form filled out by at least 1 representative of group/project
- [ ] Send custom metadata template spreadsheet, how-to guide and directions for submitting data, matrix and cell annotation files.
- [x] Receive spreadsheet (to a sufficient standard to submit dataset) 
- [x] Receive data, matrix and cell annotation files (to a sufficient standard to submit dataset)
- [x] Curate metadata spreadsheet with ontologies
- [x] Upload sheet to validate metadata
- [x] Run graph validator & Check linking
- [x] Validate data files
- [x] Ask the Secondary Wrangler for an end-to-end review of the project. Ask the Expertise Wrangler to review specific tabs if needed
- [ ] Get final approval of submission from the contributor including release date, metadata spreadsheet and data files 
- [x] Submit to production and inform ingest-devs the project is ready for archiving
- [ ] Convert project data to SCEA format following the SCEA conversion SOP if appropriate

**Please track the below as well as the key events:**
1. Track date first spreadsheet received and final spreadsheet sent by editing ticket to include
date next to event.
1. Track spreadsheet iterations by placing asterisks next to receive spreadsheet event.
1. Track any metadata issues/tickets made for dataset with a bulleted list of links under
received spreadsheet event. Links should be to the ticket in the metadata repo.

**Comment by Wkt8** [comment]
This is waiting on the project accession to become public, which should happen in a week.

**Comment by Wkt8** [comment]
The project accession is now public, and will begin wrangling.

**Comment by Wkt8** [comment]
This project is now ready for secondary review. 
Project is here: https://staging.contribute.data.humancellatlas.org/submissions/detail?id=606c4377ed3471456300379a&project=037673f8-323f-416e-92ef-c76c8e215b2f

There is an Ingest-graph-validator issue: 
`21-04-06 12:14:39 [ingest_graph_validator.actions.test_action] - ERROR: test [file_donthave_outward_links.adoc] failed: non-empty result.
21-04-06 12:14:39 [ingest_graph_validator.actions.test_action] - ERROR: result: [{'f': (_381:analysis_file:file {describedBy: 'https://schema.humancellatlas.org/type/file/6.2.0/analysis_file', `file_core.content_description`: "[{'text': 'h5ad file  for BCR enriched libraries, TCR enriched lirbaries, cell surface proteins, and gene expression data', 'ontology': 'data:2082', 'ontology_label': 'Matrix'}]", `file_core.file_name`: 'covid_portal.h5ad', `file_core.format`: 'h5ad', id: 'covid_portal.h5ad', schema_type: 'file'})}, {'f': (_382:analysis_file:file {describedBy: 'https://schema.humancellatlas.org/type/file/6.2.0/analysis_file', `file_core.content_description`: "[{'text': 'csv file with barcodes and annotated  cell types and cell surface proteins', 'ontology': 'data:2655', 'ontology_label': 'Cell type identifier'}]", `file_core.file_name`: 'annotation_200112.csv', `file_core.format`: 'csv', id: 'annotation_200112.csv', schema_type: 'file'})}]`

Looking at the test itself (https://github.com/ebi-ait/ingest-graph-validator/blob/master/graph_test_set/file_donthave_outward_links.adoc) and looking at the specific files in the ingest-graph-validator, there are no outward links from the analysis_file that link outwards, so I'm not sure what the problem is unless it's taking the 'Project' link and failing because of that. 

![graph (1).png](https://images.zenhubusercontent.com/5fd74f67b86eb46030b3f4e3/4bdc78b0-f5dd-4415-8c72-06d9e3d5181e)

**Comment by Wkt8** [comment]
Updating to say that this error 'file_donthave_outward_links' is also found in many other projects, and is also caused by sequence files which do not have outward links (via a process) to other entities. Branching off this issue to a separate ticket to look at it, but it's not essential at the moment (unless secondary reviewer thinks otherwise!)

**Comment by mshadbolt** [comment]
Wei and I had a discussion about some queries I had when secondary reviewing. Overall it was really good and he has contacted the contributor for a few extra details.

**Comment by mshadbolt** [comment]
Just noticed this is now published here: https://www.nature.com/articles/s41591-021-01329-2

**Comment by rays22** [comment]
There is a peer-reviewed paper that looks related to this dataset and says

> The processed data are available to download from Array Express under accession number E-MTAB-10026.

Stephenson E, Reynolds G, Botting RA, et al. Single-cell multi-omics analysis of the immune response in COVID-19. Nature Medicine. 2021 Apr. DOI: 10.1038/s41591-021-01329-2 PMID: 33879890 

> The processed data are available to download from Array Express under accession number E-MTAB-10026.

The authors of the pre-print and the peer-reviewed paper overlap but are not identical. The title of the pre-print and the peer-reviewed paper also differ, but both publications seem to be based on the same primary data.

**Comment by Wkt8** [comment]
Hi @rays22 do you have a link to the peer-reviewed paper? Is it the same link as Marion posted above 'https://www.nature.com/articles/s41591-021-01329-2'?

**Comment by rays22** [comment]
Yes, it is the same paper: <https://doi.org/10.1038/s41591-021-01329-2>

**Comment by Wkt8** [comment]
I have been in correspondence with the authors and sent them the final spreadsheet (including the project link to the peer-reviewed paper) and all the necessary metadata. I am fairly certain they are both the same paper - just that they have not updated the pre-print with the peer-reviewed link.

**Comment by Wkt8** [comment]
The EGA submission was successful with accession : EGAS00001005465

The following updates need to be made:
- add the EGA accession to the project field
- update the analysis protocol ontology with the new ontologies

**Comment by Wkt8** [comment]
Added EGA Accession and analysis protocol ontology (processed_matrix_generation) just need to export

**Comment by ESapenaVentura** [comment]
The analysis protocol ontologies are not exposed anywhere in the DCP so it's virtually impossible to check

The EGA accession has been added successfully and the number of files seems normal.

Moving it to verified, let me know if you'd like to move it back
